backend/lucene/src/main/java/org/hibernate/search/backend/lucene/search/projection/impl/FieldSearchProjectionBuilderImpl.java

```diff
@@ -11,7 +11,7 @@
 import org.hibernate.search.engine.search.SearchProjection;
 import org.hibernate.search.engine.search.projection.spi.FieldSearchProjectionBuilder;
 
-public class FieldSearchProjectionBuilderImpl<T> implements FieldSearchProjectionBuilder<T> {
+public class FieldSearchProjectionBuilderImpl<T, U> implements FieldSearchProjectionBuilder<U> {
```

Comment:
Shouldn't `T` be renamed to `F`? From what I understand it's the actual field type, which is named `F` everywhere else, including in the codec.

If I understand correctly, `U` is not the same as the type `T` in the converter, though (it the type requested by the user, not the type used internally when querying).

Reply:
Right, done.